full-stack-tests/backend/src/integration/Checkpoints.test.ts

```diff
@@ -32,14 +32,14 @@ describe.skip("Checkpoints", () => {
   let testITwinId2: GuidString;
   let testChangeSet2: ChangesetProps;
 
-  const blockcacheDir = path.join(KnownTestLocations.outputDir, "blockcachevfs");
```

Comment:
@wgoehrig all of these tests are currently "skipped" (see above). 

There is some problem starting/stopping the daemon that hangs occasionally. I think there should be only one test that starts the daemon, tests everything, and then stops the daemon rather than starting/stopping in before/after methods.

Reply:
@kabentley - I just pushed a change to reorganize the tests a bit and only start/stop the daemon once - does that look good to you?